sdks/python/apache_beam/utils/urns.py

```diff
@@ -26,11 +26,11 @@
 from apache_beam.utils import proto_utils
 
 
-PICKLED_WINDOW_FN = "beam:window_fn:pickled_python:v0.1"
-GLOBAL_WINDOWS_FN = "beam:window_fn:global_windows:v0.1"
-FIXED_WINDOWS_FN = "beam:window_fn:fixed_windows:v0.1"
-SLIDING_WINDOWS_FN = "beam:window_fn:sliding_windows:v0.1"
-SESSION_WINDOWS_FN = "beam:window_fn:session_windows:v0.1"
+PICKLED_WINDOW_FN = "beam:windowfn:pickled_python:v0.1"
+GLOBAL_WINDOWS_FN = "beam:windowfn:global_windows:v0.1"
```

Comment:
We had discussed using a proto definition like

```
message WellKnownUrns {
  string global_windows = 1 [default = "urn:beam:windowfn:global_windows:0.1"]
  ...
}
```

But I guess this is proto 2 only. For normal proto use, I agree with this change, but it is too bad we don't have that hack available. Any other ideas? A data file seems mostly pointless because whatever you look up the URN by is effectively just another unique name anyhow (it would probably prevent _some_ errors).

Not that I think there is a huge danger of these getting out of sync, since they can never change once they are really used. But it would be nice for new SDKs, etc.